code/src/components/GamePage.js

```diff
@@ -0,0 +1,183 @@
+    return (
+      <article className="GamePageContainer">
+        <section className="GamePageMapContainer">
+          <img className="GampePageMap" src={map} alt="map" />
+
+          <div className="LoadingContent">
+            <div className="LoadingAnimation">
+              <Lottie
+                options={defaultOptions}
+                height={200}
+                width={200}
+                style={{
+                  position: "absolute",
+                }}
+              />
+            </div>
+          </div>
+        </section>
+
+        <section className="DirectionMap">
+          <div className="map-grid">
+            <Lottie
+              className="LottiMap"
+              options={defaultOptionsWater}
+              style={{
+                height: "30vh",
+                width: "96vw",
+                position: "absolute",
+              }}
+            />
+
+            {createMapTiles()}
+          </div>
+        </section>
+      </article>
+    );
+  } else if (!loading) {
+    return (
+      <article className="GamePageContainer">
+        <section className="GamePageMapContainer">
+          <img className="GampePageMap" src={map} alt="map" />
+          <div className="LoadingContent"></div>
+          <section className="GamePageContent">
+            <section className="GamePageStory">
+              <div className="GamePageCoordinates">
+                <div
+                  height={40}
+                  width={40}
+                  style={{
+                    position: "relative",
+                    right: "45vw",
+                    display: "block",
+                    marginTop: "-5px",
+                  }}
+                >
+                  <Lottie options={defaultOptions} height={40} width={40} />
+                </div>
+                <p> coordinates:{labyrinth.coordinates}</p>
+              </div>
+              <p>{labyrinth.description}</p>
+            </section>
+
+            <section className="GamePageItemContainer">
+              {labyrinth.actions.map((item) => (
+                <div className="GamePageItem">
+                  <Direction
+                    direction={item.direction}
+                    key={item.description}
+                    description={item.description}
+                    directions={item.direction}
+                    setdDirection={setDirection}
+                  />
+                </div>
+              ))}
+            </section>
+          </section>
+        </section>
+
+        <section className="DirectionMap">
+          <div className="map-grid">
+            <Lottie
+              className="LottiMap"
+              options={defaultOptionsWater}
+              style={{
+                height: "30vh",
+                width: "96vw",
+                position: "absolute",
+              }}
+            />
+
+            {createMapTiles()}
+          </div>
+        </section>
+      </article>
+    );
+  }
```

Comment:
We suggest, that you make two components to use in your if/else statement. Like this your code will look cleaner and be easier to read. `{loading ? <LoadingAnimation /> : <GamePageAnimation />}`